generated/api/v1/cluster_service.swagger.json

```diff
@@ -263,6 +270,90 @@
         }
       }
     },
+    "ClustersServicePutClusterBody": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "title": "@gotags: search:\"Cluster,store\" sql:\"unique\""
```

Comment:
This is something we didn't expect

Reply:
![image](https://github.com/stackrox/stackrox/assets/1616386/7d02d031-29c5-4610-95dc-c3856b38611d)